src/osmf/js/osmf_flash.js

```diff
@@ -547,7 +570,10 @@ require("../../../html5-common/js/utils/constants.js");
     };
 
     var raiseVolumeEvent = function(event) {
-      newController.notify(newController.EVENTS.VOLUME_CHANGE, { "volume" : event.eventObject.volume });
+      var volume = event.eventObject.volume;
+      var muted = volume === 0;
+      newController.notify(newController.EVENTS.VOLUME_CHANGE, { "volume" : volume});
+      newController.notify(newController.EVENTS.MUTE_STATE_CHANGE, { muted : muted });
```

Comment:
Could there be any issues if this is raised with the same mute state as the previous event?

Reply:
That's up to each plugin in how they consume it, but from testing I have not observed anything. For SDKs where we cannot get the muted state, we would have to maintain it, sometimes without the proper hooks. This might lead to times where we don't send the proper mute state. I think sending out the mute states whenever we receive volume events from the SDK is the safer vs sometimes gating mute state events. What do you think?

Reply:
Oh ok, sounds good